schemas/extractor.yml

```diff
@@ -0,0 +1,74 @@
+---
+name: extractor
+id: https://www.marda-alliance.org/extractors/extractor
+prefixes:
+    linkml: https://w3id.org/linkml/
+    schema_org: http://schema.org/
+    dublin_core: http://purl.org/dc/elements/1.1/
+imports:
+    - linkml:types
+default_range: string
+description: >-
+    This file describes the `Extractor` model created by the MaRDA extractors WG.
+
+
+classes:
+    Extractor:
+        description: >-
+            A code object or web service that when executed can read a file with specific
+            file type, or a
+            set of files with a given structure, and extract or transform the scientific
+            data
+            and/or metadata contained within the file.
```

Comment:
Formatting here is a bit iffy.

Reply:
fixed, but pre-commit-ci might decide to "fix" it back...